lib/tools/buildScanner.js

```diff
@@ -0,0 +1,40 @@
+// Module for scanning the .build directory and alerting the user when they have stale files
+require('colors')
+const fs = require('fs')
+const path = require('path')
+const klawSync = require('klaw-sync')
+
+module.exports = function (app) {
+  const logger = require('./logger')(app.get('params').logging)
+  let buildFiles
+  let compDir = app.get('params').js.output.split('/')[0]
```

Comment:
The scanner does technically account for built css as well as `app.get('params').js.output` returns the string `.build/js` and then the `split('/')[0]` appended to the end of the statement gives us `.build`. 
I did this to avoid hardcoding `.build` in there as it is configurable.
I just wanted to let you know that, so I can clear up exactly what is needed. I could proceed by writing a similar statement such as `app.get('params').css.output.split('/')[0]`, which would be needed in the case that the user has different `.build` directories for their css and js, if you think considering for that case is necessary.

Reply:
Scratch what I had said before, the solution I just pushed up should be what you want.